src/components/item/publish/CCLicenseSelection.js

```diff
@@ -115,24 +123,42 @@ const CCLicenseSelection = ({ item }) => {
           value={CC_LICENSE_ADAPTION_OPTIONS.ALLOW}
           control={<Radio color="primary" />}
           label={t('Yes')}
-          disabled={disabled}
         />
         <FormControlLabel
           value={CC_LICENSE_ADAPTION_OPTIONS.ALIKE}
           control={<Radio color="primary" />}
           label={t('Only if others share alike')}
-          disabled={disabled}
         />
       </RadioGroup>
       <CCLicenseDialog
         open={open}
         setOpen={setOpen}
-        disabled={disabled || !optionValue}
+        disabled={!optionValue}
         className={classes.button}
+        buttonName={t('Submit')}
+        dialogContent={t(
+          'Please verify that your item fits the CC License, and do not change to a more restricted option.',
+        )}
         handleSubmit={handleSubmit}
       />
+      {settings?.ccLicenseAdaption !== '' && (
+        <CCLicenseDialog
+          open={open}
+          setOpen={setOpen}
+          disabled={!optionValue}
+          className={classes.button}
+          buttonName={t('Remove License')}
+          dialogContent={t(
+            'Are you sure to remove existing CC License? This action is not suggested, and may lead to license inconsistency.',
+          )}
+          handleSubmit={handleRemove}
```

Comment:
You didn't provide a screenshot, but I suppose removing the license is a button. Would it make sense to have it in the options instead? Something like `None`?